docker/cloudhsm-entrypoint.sh

```diff
@@ -5,29 +5,29 @@
 
 if [[ -z "$HSM_IP" ]]; then
   #statements
-  echo -e "\n* HSM_IP env variable must be set \n"
-  exit 1
-fi
-
-# Configure cloudhms-client
-# COPY customerCA.crt /opt/cloudhsm/etc/
-/opt/cloudhsm/bin/configure -a $HSM_IP
+  echo -e "\n* HSM_IP env variable must be set to allow CloudHSM connectivity \n"
```

Comment:
```suggestion
  echo -e "\n* HSM_IP env variable not set: skipping CloudHSM connection \n"
```
more descriptive maybe of the actual flow happening?